provision/provisiontest/fake_provisioner.go

```diff
@@ -1068,6 +1068,36 @@ func (p *FakeProvisioner) ExecuteCommandOnce(stdout, stderr io.Writer, app provi
 	return nil
 }
 
+func (p *FakeProvisioner) ExecuteCommandIsolated(stdout, stderr io.Writer, app provision.App, cmd string, args ...string) error {
```

Comment:
Could you add a simple test for this function in fake_provisioner_test.go? Similar to https://github.com/tsuru/tsuru/blob/master/provision/provisiontest/fake_provisioner_test.go#L871. Just to make sure this code is executed during tests.